### ISSUE TITLE
Style "Submit" button on "Contact Me" form

### ISSUE THREAD

**Issue by vikascripps** [mwkwsd/sensensomething#165]
Currently the button looks like this in staging:
<img width="100" alt="Screen Shot 2024-05-06 at 8 57 51 AM" src="https://github.com/mwkwsd/sensensomething/assets/91079058/81888a6b-9890-4f9a-b0be-90d920417fd0">

It should look like this:
![Screen Shot 2024-05-06 at 8 58 51 AM](https://github.com/mwkwsd/sensensomething/assets/91079058/956f712c-4476-4032-b088-242be8d84b59)

I mentioned the font update for button text in #164. It probably is better to update text style for all buttons at once. 
In addition to text style change for this button, we also need to update the styling of the box. Here is a [link to this button in Figma](https://www.figma.com/file/Sw7uUdfgvrYhLAyIqU3ERK/SensenSomething?type=design&node-id=1977%3A361&mode=design&t=5p5lZCZFrDE8C0Ft-1).

Here are specs for it:
<img width="240" alt="Screen Shot 2024-05-06 at 9 03 42 AM" src="https://github.com/mwkwsd/sensensomething/assets/91079058/20b9a6e0-5130-4670-b214-4b01ca56b43d">

**Comment by mwilkins74** [comment]
See #156. 
I have been unable to get this button do any restyling other than removing the rounded border edges. 
I have spent at least two hours trying to "expand" the button so there is more space on either side of the Submit label to no avail. 

@ChuckBTaylor any suggestions?

**Comment by mwilkins74** [comment]
@vikascripps 
Ok, after like 3 hours of wrestling this damn button I've gotten it to look like this! *phew*

<img width="687" alt="Screen Shot 2024-05-10 at 2 59 01 PM" src="https://github.com/mwkwsd/sensensomething/assets/102488171/1583a690-aa96-4a89-bc9e-8d9fb4a722c7">

**Comment by vikascripps** [comment]
@mwilkins74 yay, great job! Sorry it took so long!
The text style on the button is wrong but I'm assuming it will be handled in a different issue.

**Comment by mwilkins74** [comment]
@vikascripps 
It's using the Manrope font-family as you can see in this screenshot.
Let me know if that needs to be changed to something else. 

<img width="346" alt="Screen Shot 2024-05-10 at 4 52 36 PM" src="https://github.com/mwkwsd/sensensomething/assets/102488171/29697fca-d64d-4045-aa7d-1a9534399627">

**Comment by vikascripps** [comment]
@mwilkins74 ha, I'm not sure what is happening, but the font on your screenshot is not Manrope. I understand that that's what it says when you inspect it, but this is not what Manrope looks like.

Here is what it looks like:
![Screen Shot 2024-05-10 at 3 12 32 PM](https://github.com/mwkwsd/sensensomething/assets/91079058/a926d85a-ab35-4bbb-99e6-d40ebc272243)

Here is a[ link to it in Google Fonts](https://fonts.google.com/specimen/Manrope).

**Comment by mwilkins74** [comment]
@vikascripps Very weird!

Any chance the font we're seeing on render is `Oswald` or `Roboto`?

We have those declared in the application and I have a feeling `Oswald` may be what we're seeing even though it's telling me it's Manrope.

**Comment by vikascripps** [comment]
@mwilkins74 no, it's not either of those. They are both sans serif fonts and the one that's rendering is a serif.

**Comment by mwilkins74** [comment]
@vikascripps so Weird!!!
I'll do some more exploring. 

p.s. What's a serif?

**Comment by vikascripps** [comment]
@mwilkins74 serif fonts have those decorative lines or tapers (also commonly referred to as “tails” or “feet”) while sans serif fonts don't—hence the “sanes” in their title. 

So a serif would look like this (I circled the "tails" mentioned above):
![Frame 1009](https://github.com/mwkwsd/sensensomething/assets/91079058/7dd7f3be-75cc-4469-a829-df1e225e1d41)

And a sans serif would look like this (it's very straight and does not have "tails"):
![Screen Shot 2024-05-14 at 10 26 56 AM](https://github.com/mwkwsd/sensensomething/assets/91079058/65b4dfad-af76-4a45-a399-a4b772dd97d5)

**Comment by mwilkins74** [comment]
@vikascripps I left this hanging for a couple of weeks, but I finally think I figured it out. 
This is the current `SUBMIT` button styling. 
When I inspect the element it still says `Manrope`, but now it includes a `sans-serif` flag as well. 
I think I needed to import the font into the code, which once I did this is the result. 
<img width="430" alt="Screen Shot 2024-05-28 at 6 06 53 PM" src="https://github.com/mwkwsd/sensensomething/assets/102488171/843aad5f-324f-41e0-80b4-b85d6a9bcde7">

**Comment by vikascripps** [comment]
@mwilkins74 Nice, the font looks great!
I seems that the button now moved to the right, could you right align in with the right end of the 'Message' input field please?
Also, the button now has rounded corners, could you make them sharp?

**Comment by mwilkins74** [comment]
@vikascripps good morning!
This is current Submit button when I spin it up this morning. 
Let me know further thoughts. 

<img width="360" alt="Screen Shot 2024-06-24 at 8 49 46 AM" src="https://github.com/mwkwsd/sensensomething/assets/102488171/31856948-8468-4e2f-a875-1d5705f1f943">

**Comment by vikascripps** [comment]
@mwilkins74 @ChuckBTaylor @kmdunn5 tagging all of you as I am not sure who can work on this.
The latest screenshot has the outline of the button looking right, but the font of the button copy is wrong.

The website has the font looking right but the outline looks wrong:
![Image](https://github.com/user-attachments/assets/6e4b0d82-8d09-4fe9-98b3-6a8356310d99)

[Here is the button on Figma](https://www.figma.com/design/Sw7uUdfgvrYhLAyIqU3ERK/SensenSomething?node-id=1852-2699&m=dev)

Here are the specs it should have:

Layout:
display: flex;
padding: 12px 24px;
align-items: center;
gap: 10px;

Style: 
border: 1px solid #FFB74D;

Text Style:
color: #FFB74D;
font-family: Manrope;
font-size: 12px;
font-style: normal;
font-weight: 500;
line-height: normal;
letter-spacing: 0.24px;
text-transform: uppercase;
